spec/swagger/swagger_doc_v2.yaml

```diff
@@ -719,3 +719,45 @@
             application/vnd.api+json:
               schema:
                 "$ref": "../v2/post_event_responses.yaml#/422"
+  "/categories":
```

Comment:
can you also update the documentation to include the new endpoint for getting a single person?

Reply:
Swagger docs updated to include `GET /people/{person_id}` 👍